src/components/Node/Node.js

```diff
@@ -244,20 +245,23 @@ export default function Node({
           )}
         </Button>
       )}
-      {node.data.downIds && !node.data.downIds.length && node.data.childrenCount && node.data.childrenCount > 0 && currentProp && currentProp.id === CHILD_ID && (
+      {node.data.downIds &&
+        !node.data.downIds.length &&
+        !!node.data.childrenCount &&
```

Comment:
you have to cast to bool numbers otherwise you get this 
<img width="410" alt="Screenshot 2020-10-19 at 22 29 08" src="https://user-images.githubusercontent.com/4820803/96508999-56a26180-1253-11eb-9ca5-0f83490ed9a0.png">

Reply:
Thanks I kept wondering why, but where did you cast it now, probably the !! already catches it